docs/docs/run-node/01-run-node.md

```diff
@@ -149,3 +149,15 @@ In config.toml:
 ```toml
 log_level: "state:info,p2p:info,consensus:info,x/staking:info,x/ibc:info,*error"
 ```
+
+## State Sync
+
+State sync is the act in which a node syncs the latest or close to the latest state of a blockchain. This is useful for users who don't want to sync all the blocks in history. You can read more here: https://docs.cometbft.com/v0.37/core/state-sync
+
+### Local State Sync
+
+Local state sync work similar to normal state sync except that it works off a local snapshot of state instead of one provided via the p2p network. The steps to start local state sync are similar to normal state sync with a few different designs. 
+
+1. As mentioned in https://docs.cometbft.com/v0.37/core/state-sync, one must set a height and hash in the config.toml along with a few rpc servers (the afromentioned link has instructions on how to do this), but you must leave state sync set to false, this is crucial for when you start your node. 
```

Comment:
> but you must leave state sync set to false

I think it's ok to set `enable` to whatever value, because it'll be ignored if the state is not empty.